cloud/azure/modules/app/outputs.tf

```diff
@@ -1,15 +1,15 @@
 output "app_service_default_hostname" {
-  value = "https://${azurerm_app_service.civiform_app.default_site_hostname}"
-}
-
-output "app_service_canary_hostname" {
-  value = "https://${azurerm_app_service_slot.canary.default_site_hostname}"
+  value = "https://${azurerm_linux_web_app.civiform_app.name}.azurewebsites.net"
```

Comment:
**question:** This looks like it'll need to be changed back to allow general URLs - what is the purpose of including the change here?

Reply:
`default_site_hostname` is no longer an attribute in the newer version of the resource `azurerm_linux_web_app` (as compared to the deprecated resource `azurerm_app_service`) but I just realized `default_hostname`.